src/winetricks

```diff
@@ -718,6 +718,26 @@ w_expand_env()
     winetricks_early_wine cmd.exe /c echo "%$1%"
 }
 
+# Check if $1 is executable (output custom message using w_debug if $2 is specified) and return 126 (standart non-executable exit code) or 0 if executable
+w_check_exec()
+{
+    w_executable="$1"
+    w_message="$2"
+
+    if ! command -v "$w_executable" 2>/dev/null; then
+        if [ -n "$w_message" ]; then
+            printf '%s\n' "$w_message"
+            return 126
+        elif [ -z "$w_message" ]; then
+            printf '%s\n' "Command $w_executable is not executable"
+            return 126
+        fi
+    elif command -v "$w_executable" 2>/dev/null; then
```

Comment:
You don't need to run command again; either passed or it failed the first time.

Reply:
`else` is going to trigger on anything unexpected where provided logic might return unexpected on non-standard system -> Reasoning to use `elif`

Or do you want me to use `else` anyway?

Reply:
Unless you have an actual example of what something unexpected could be, yes, use else.

In other words, it's either executable or it's not.